### DISCUSSION TITLE
Create private cloud instance and connect Qfield to it

### DISCUSSION THREAD

**Discussion by Celpho** [opengisch/QField#2927] [Q&A]
Hello everyone,

I'm trying to create a private cloud instance for my work in order to speed up the sync (we have a large database).

I've managed to create my container, migrate the database and create a super user (I've made all steps in the readme for "launch a local instance")  but i don't know after that how can i communicate with the API in order to create a project with some user accounts and put my Qgs_project file on it. 

I've tried to open http://localhost:8000/ in my browser but i get the "Err_connection_refused"

Which commands should i use to communicate with the api and do basics stuff? (I've checked the doc here : https://app.qfield.cloud/swagger/  and here :  https://app.qfield.cloud/docs/     which explain the commands but i don't know exactly what i have to type before to make these commands working)

Also after that i will have to change the URL called in qfield to make the app communicate with the api. Which lines do i have to change in the code to make the app connect to my private API and how can i install it on my android ? 

Sorry if my questions are stupid, i'm a newbie in programming.

**Comment by suricactus** [comment]
Hey @Celpho . Please explore the https://github.com/opengisch/qfieldcloud-sdk-python for examples and a client that works.

On QGIS and QField, double tap the QFieldCloud logo to change the API URL to your own instance.

**Reply by Celpho** [reply]
Thank you for your answer :)
I've few more questions not directly related to this topic but i will search a little on my own before creating an other topic.
